CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for stripes-core
 
+## [3.11.1] (In Progress)
```

Comment:
I believe the next version should really be 3.12.0. 

cc @zburke

Reply:
True, but the value in the CHANGELOG doesn't have any practical effect so I'm not worried about it. This could merge to a `b3.11` branch if we publish any patches, and/or it might also merge to master and be released as part of the next feature release, i.e. `v3.12.0`.